ubuntu-dev/workspace-janitor.js

```diff
@@ -71,6 +71,21 @@ module.exports = function (options) {
 
           // Use a longer scrollback for the Terminal.
           p.settings.user.terminal['@scrollback'] = 10000;
+
+          if (!p.settings.project) {
```

Comment:
Nit: Please add a comment similar to "Tweak the default user settings" above (ideally with a link to a file or list of every available project setting).